src/server.rs

```diff
@@ -397,8 +397,14 @@ impl<O: Output> LsService<O> {
         self.output.success(id, ResponseData::Init(result));
 
         let root_path = params.root_path.map(PathBuf::from).expect("No root path");
+        let init_options: Option<InitializationOptions> = params.initialization_options
+                              .and_then(|options| serde_json::from_value(options)
+                                            .unwrap_or_else(|err| {
+                                                debug!("Error parsing initialization_options: {:?}", err);
+                                                None
+                                            }));
         self.handler.init(root_path);
-        self.handler.inited().init(self.output.clone());
+        self.handler.inited().init(init_options, self.output.clone());
```

Comment:
Not passing `init_options` to `.init()` since it'd have to store some state to be useful for `.inited().init()` and the only supported option right now will be only used once on init, which means it isn't probably worth keeping around. Is that okay for now?